### PR TITLE
[WebRTC] Change Stats enum value and add new enum value

### PR DESCRIPTION
**Pull request by hsgwon**
### Description of Change ###

- Change Stats enum value
- Add new value in WebRTCStatisticsCategory enum

### API Changes ###

 - TCSACR-535

### CODE REVIEW

**Comment by hsgwon** [comment]
API 변경에 대한 결재 진행 중.

**Review by aferin — APPROVED** [review]

**Comment by hsgwon** [comment]
> API 변경에 대한 결재 진행 중.

결재 완료. ACR페이지에 결재 메일 첨부

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs [hunk 1]
```diff
@@ -546,137 +543,197 @@ public enum WebRTCStatisticsProperty
...
+        Kind,
+
+        /// <summary>
+        /// Address of the candidate.
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        Address,
 
         /// <summary>
         /// Local ID.
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:691]: that summary is attached to the Port property, the Local ID is at line 674

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs [hunk 2]
```diff
@@ -510,31 +510,28 @@ public enum WebRTCStatisticsCategory
...
+        /// </summary>
+        /// <since_tizen> 11 </since_tizen>
+        RemoteCandidate = 0x0080,
 
-        RemoteOutboundRtpStream = 0x00010000
+        /// <summary>
+        /// All types of WebRTC statistics.
+        /// </summary>
+        [Obsolete("Deprecated since API11; Will be removed in API13.")]
+        All = Codec | InboundRtp | OutboundRtp | RemoteInboundRtp | RemoteOutboundRtp | CandidatePair | LocalCandidate | RemoteCandidate
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:534]: changing of existing enum values are not normally allowed, an argument can be made that this holds the purpose of the "All" unchanged

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs [hunk 3]
```diff
@@ -546,137 +543,197 @@ public enum WebRTCStatisticsProperty
         /// <summary>
         /// Timestamp.
         /// </summary>
-        Timestamp = WebRTCStatisticsPropertyCategory.Common | 0x01,
```
Comment (wiertel) [inline src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:549]: changing the enum values are normally allowed, I will approve the PR only after confirming it is ok

**Review by wiertel — CHANGES_REQUESTED** [review]

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs (hunk 2 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:534]: I've got approval for change from PM part.

File: src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs (hunk 1 above)
Comment (hsgwon) [reply src/Tizen.Multimedia.Remoting/WebRTC/WebRTCEnums.cs:691]: I'll fix it.

**Review by wiertel — APPROVED** [review]

**Pull request merged**